src/hotspot/share/gc/parallel/psCardTable.hpp

```diff
@@ -42,12 +42,16 @@ class PSCardTable: public CardTable {
     verify_card       = CT_MR_BS_last_reserved + 5
   };
 
+  static const size_t num_cards_in_stripe = 128;
+  static size_t large_obj_arr_min_words() { return 2 * num_cards_in_stripe * _card_size_in_words + 1; }
```

Comment:
given that the term `num_cards_in_stripe * _card_size_in_words` is calculated 3 times, it may be worth factoring it out; also maybe, instead of repeated checks `obj_size < large_obj_arr_min_words()` a predicate like `is_large_array` or something may be nicer.